frontend/apps/filemanager/filemanager.lua

```diff
@@ -377,6 +377,10 @@ function FileManager:registerKeyEvents()
         self.file_chooser.key_events.Back = { { Device.input.group.Back } }
         if Device:hasScreenKB() then
             self.key_events.ToggleWifi = { { "ScreenKB", "Home" } }
+        elseif Device:hasKeyboard() then
+            self.key_events.ShowFileSearch = { { "Shift", "S" }, event = "ShowFileSearch" }
+            self.key_events.ShowWikipediaLookup = { { "Shift", "W" }, event = "ShowWikipediaLookup" }
+            self.key_events.ShowDictionaryLookup = { { "Shift", "D" }, event = "ShowDictionaryLookup" }
```

Comment:
Why do you duplicate these in FM and in the modules?

Reply:
mmm, so they can also be accessed from the file manager too. I have been known to make unrelated (to any ebooks) wikipedia searches from there.

Reply:
Can you try without these lines in FM?

Reply:
oooh, why did that work? I thought FM and Reader were independent.